<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMMSettings</class>
 <widget class="QDialog" name="DialogMMSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>DB Server:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtServer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>DB:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtDataBase"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtUser"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pwd</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtPwd">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Gig</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinAutoCustom">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>CFR:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinCrf">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>51</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Preset:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cboxPreset"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btnSelPath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="cboxDelete">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="cboxStrip">
     <property name="text">
      <string>Strip Subs</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QCheckBox" name="cbAutoProcess">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="btnDirToMonitor">
     <property name="text">
      <string>MM Settings</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
